src/middleware/middleware.js

```diff
@@ -174,11 +174,16 @@ middleware.buildHeader = function(req, res, next) {
 			},
 			footer: function(next) {
 				app.render('footer', {loggedIn: (req.user ? parseInt(req.user.uid, 10) !== 0 : false)}, next);
+			},
+			plugins: function(next){
+				plugins.fireHook('filter:header.build', res.locals, next);
```

Comment:
Can you change this to 

`plugins.fireHook('filter:header.build', {locals: res.locals}, next);`